examples/Icequake_Iceland/iceland_lut.py

```diff
@@ -7,32 +7,24 @@
 from pyproj import Proj
 
 from QMigrate.io import read_stations
-from QMigrate.lut import compute, LUT
+from QMigrate.lut import compute_traveltimes
 
 station_file = "./inputs/iceland_stations.txt"
-data_in = "./inputs/mSEED"
-lut_out = "./outputs/lut/icequake.LUT"
-out_path = "./outputs/runs"
-run_name = "icequake_example"
+lut_out = "./outputs/lut/example.LUT"
 
 # --- Read in the station information file ---
 stations = read_stations(station_file)
 
-# --- Define projections ---
-cproj = Proj(proj="longlat", ellps="WGS84", datum="WGS84", no_defs=True)
-gproj = Proj(proj="lcc", lon_0=-17.224, lat_0=64.328, lat_1=64.32, lat_2=64.335,
-             datum="WGS84", ellps="WGS84", units="m", no_defs=True)
-
-# --- Create new LUT ---
-# Cell count (x,y,z); cell size (x,y,z in metres)
-lut = LUT(ll_corner=[-17.24363934275664, 64.31947715407385, -1390.],
-          ur_corner=[-17.204348515198255, 64.3365202025144, 1390],
-          cell_size=[100., 100., 20.], grid_proj=gproj, coord_proj=cproj)
+# --- Define the grid specifications ---
+grid_spec = {"ll_corner": [-17.24363934275664, 64.31947715407385, -1.390],
+             "ur_corner": [-17.204348515198255, 64.3365202025144, 1.390],
+             "cell_size": [0.1, 0.1, 0.02],
+             "grid_proj": Proj(proj="lcc", lon_0=-17.224, lat_0=64.328,
+                               lat_1=64.32, lat_2=64.335, datum="WGS84",
+                               ellps="WGS84", units="km", no_defs=True),
+             "coord_proj": Proj(proj="longlat", ellps="WGS84", datum="WGS84",
+                                no_defs=True)}
```

Comment:
Might actually be cleaner if we do this using an `AttribDict`:

```python
from obspy.core import AttribDict
grid_spec = AttribDict()
grid_spec.ll_corner = [-17.24363934275664, 64.31947715407385, -1.390]
...
etc.
```

It's super handy.

Reply:
Do they unpack? I do like `.` access, so would be in favour if it works as expected